### PR TITLE
[CCTRI-3295] - Implement "Events per day" tile 

### PR DESCRIPTION
**Pull request by TarasMaliuchenko**

### CODE REVIEW

File: code/api/dashboard.py [hunk 1]
```diff
@@ -13,40 +11,18 @@
 @dashboard_api.route('/tiles', methods=['POST'])
```
Comment (oshynk) [inline code/api/dashboard.py:11]: Why you do not have /tiles/tile endpoint? It's not needed?

Comment (TarasMaliuchenko) [reply code/api/dashboard.py:11]: Didn't find using it on UI. Just in case, implemented - 1061f3d. Thank you

**Review by oshynk — DISMISSED** [review]

File: code/api/tiles/__init__.py [hunk 2]
```diff
@@ -1 +1,2 @@
 from .affected_instances import AffectedInstances  # noqa: F401
+from .events_per_day import EventsPerDay           # noqa: F401
```
Comment (mstoro) [inline code/api/tiles/__init__.py:2]: It seems we have agreed to use absolute imports.

Comment (TarasMaliuchenko) [reply code/api/tiles/__init__.py]: Fixed - d152eba. Thank you.

**Review by mstoro — APPROVED** [review]

**Review by oshynk — APPROVED** [review]

**Pull request merged**